packages/cli/code-pushup.config.ts

```diff
@@ -0,0 +1,87 @@
+import 'dotenv/config';
+import { join } from 'path';
+import { z } from 'zod';
+import type { CoreConfig } from '@code-pushup/models';
+import eslintPlugin, {
+  eslintConfigFromNxProject,
+} from '../../dist/packages/plugin-eslint';
+import jsPackagesPlugin from '../../dist/packages/plugin-js-packages';
+
+// load upload configuration from environment
+const envSchema = z
+  .object({
+    CP_SERVER: z.string().url(),
+    CP_API_KEY: z.string().min(1),
+    CP_ORGANIZATION: z.string().min(1),
+    CP_PROJECT: z.string().min(1),
+  })
+  .partial();
+const env = await envSchema.parseAsync(process.env);
+
+const config: CoreConfig = {
+  ...(env.CP_SERVER &&
+    env.CP_API_KEY &&
+    env.CP_ORGANIZATION &&
+    env.CP_PROJECT && {
+      upload: {
+        server: env.CP_SERVER,
+        apiKey: env.CP_API_KEY,
+        organization: env.CP_ORGANIZATION,
+        project: env.CP_PROJECT,
+      },
+    }),
+
+  plugins: [
+    await eslintPlugin(await eslintConfigFromNxProject('cli')),
+    await jsPackagesPlugin({
+      packageManager: 'npm',
+      packageJsonPath: join('packages', 'cli', 'package.json'),
```

Comment:
Invalid property name:

![image](https://github.com/code-pushup/cli/assets/34691111/e134e07e-5ed4-4962-86ee-36d3026f2efc)


```suggestion
      packageJsonPaths: [join('packages', 'cli', 'package.json')],
```

---

However, I don't think it makes sense to run JS packages plugin per project. It still runs `npm audit`, etc. on the root `package-lock.json`, so the results are the same for each project. E.g. outdated `lighthouse` is reported for `cli` project, even though `cli` doesn't use it.

Reply:
Wouldnt it help if we know per project what we release?